examples/client/convert_handle/convert_handle.go

```diff
@@ -1,24 +1,24 @@
 package main
 
 import (
-	"log"
-
 	"github.com/bitcoin-sv/go-paymail"
+	"github.com/bitcoin-sv/go-paymail/logging"
 )
 
 func main() {
+	logger := logging.GetDefaultLogger()
```

Comment:
It is only example,
I'm not sure if we should use logger here.

Reply:
Done